articles/vmware/vmw-ref-vcd97-known-issues.md

```diff
@@ -0,0 +1,153 @@
+---
+title: vCloud Director 9.7 known issues | UKCloud Ltd
+description: This article describes some of the issues you may encounter when using vCloud Director 9.7 with UKCloud for VMware
+services: vmware
+author: Sue Highmoor
+reviewer: 
+lastreviewed: 
+
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: vCloud Director 9.7 known issues
+toc_fullpath: Reference/vmw-ref-vcd97-known-issues.md
+toc_mdlink: vmw-ref-vcd97-known-issues.md
+---
+
+# vCloud Director 9.7 known issues
+
+## Overview
+
+This article describes some of the issues you may encounter when using vCloud Director 9.7 with UKCloud for VMware.
+
+## Known issues with virtual machines
+
+### Disk sizes are in MB
+
+#### Description
+
+When adding a disk in the *Hardware* section of the VM details page, you must enter disk size in MB rather than GB and values must be entered in MB.
+
+#### Solution
+
+There is currently no workaround for this issue. You must enter disk sizes in MB. We have raised a bug with VMware.
+
+### Can enter non-optimal values for disks
+
+#### Description
+
+The UI doesn't give default or suggested values for **Bus Number** or **Unit Number**, leading to the potential to enter non-optimal values.
+
+#### Solution
+
+There is currently no workaround for this issue. We have raised a bug with VMware.
+
+### Not using default storage policy when adding a new disk
+
+#### Description
+
+When adding a new disk, the storage **Policy** does not default to the VM or VDC default option.
+
+#### Solution
+
+You must change the storage policy as required.
+
+### Default option is to automatically power-on VMs
+
+#### Description
+
+When you create a VM using the Tenant UI, the **Power on** check box is selected by default so that the newly created VM is powered on. In the Legacy UI, the default was for VMs to be powered off.
+
+#### Solution
+
+If you do not want a newly created VM to be powered on, ensure that you deselect the **Power on** check box.
+
+### Cannot perform some tasks on a shut down VM, such as upgrade hardware version
+
+#### Description
+
+When you select the **Shut Down Guest OS** option for a VM leaves the VM in a partially powered off state. However in the Tenant UI, the VM will show as **Powered off**. While in this partially powered off state, you will be unable to perform tasks that require the VM to be completely powered off.
+
+#### Solution
+
+To make sure that a VM is fully powered off, select the **Power Off** option from the **Actions** menu.
+
+### Cannot select NIC type when adding a new NIC
+
+#### Description
+
+When adding a new NIC in the *Hardware* section of the VM details page, it is not possible to specify the NIC type.
+
+#### Solution
+
+<Is there a solution for this one?>
```

Comment:
still no answer?

Reply:
@mikepukcloud, as part of your review, could you confirm if there is a solution/workaround for this issue? Thanks!

Reply:
Add a solution of:
There is currently no workaround for this issue. We have raised a bug with VMware.